### PR TITLE
[16.0] [MIG] product pack

### PR DESCRIPTION
**Pull request by JulyChacko1601**

### CODE REVIEW

**Comment by nicolascol** [comment]
Hi! The runboat build is pending. could someone repair it?

**Comment by pedrobaeza** [comment]
Please cherry-pick and adapt #113

File: product_pack/models/product_pack_line.py [hunk 1]
```diff
@@ -56,4 +56,4 @@ def _check_recursion(self):
 
     def get_price(self):
         self.ensure_one()
-        return self.product_id.price * self.quantity
+        return self.product_id.lst_price * self.quantity
```
Comment (bruno-zanotti) [inline product_pack/models/product_pack_line.py:59]: lst_price in not exactly the same as price in v15, probably you should call `_get_contextual_price`() instead. Related Odoo PR: https://github.com/odoo/odoo/pull/79605

File: product_pack/models/product_product.py [hunk 2]
```diff
@@ -51,10 +51,17 @@ def split_pack_products(self):
...
         return packs, (self - packs)
 
-    def price_compute(self, price_type, uom=False, currency=False, company=False):
+    def price_compute(
+        self, price_type, uom=False, currency=False, company=False, date=False
+    ):
+        company = company or self.env.company
+        date = date or fields.Date.context_today(self)
+
+        self = self.with_company(company)
```
Comment (bruno-zanotti) [inline product_pack/models/product_product.py:60]: Is this necessary? I think it is included in the super method and it is not used after

**Review by bruno-zanotti — COMMENTED** [review]
@JulyChacko1601 
Hi July, thanks for your contribution, here I leave you some comments about the migration commit.
Could you check it?

File: product_pack/README.rst [hunk 3]
```diff
@@ -94,6 +94,7 @@ Authors
 * NaN·tic
 * ADHOC SA
 * Tecnativa
+* bloopark systems GmbH & Co. KG
```
Comment (bruno-zanotti) [inline product_pack/README.rst:97]: Hi @JulyChacko1601 as I mentioned in the migration PR of sale_product_pack, in my opinion, it is ok if you add yourself as a contributor but the authors are which ones created the module or made stronger modifications (refactors)

File: product_pack/__manifest__.py [hunk 4]
```diff
@@ -2,11 +2,12 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Product Pack",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Product",
     "summary": "This module allows you to set a product as a Pack",
     "website": "https://github.com/OCA/product-pack",
-    "author": "NaN·tic, ADHOC SA, Tecnativa, Odoo Community Association (OCA)",
+    "author": "NaN·tic, ADHOC SA, Tecnativa,bloopark systems GmbH & Co. KG, "
```
Comment (FrancoMaxime) [inline product_pack/__manifest__.py:9]: @Bruno-Zanotti If I understood correctly what you said in the author section, bloopark should be removed from this manifest key

File: product_pack/models/product_product.py (hunk 2 above)
Comment (FrancoMaxime) [reply product_pack/models/product_product.py:60]: Yes, its included in the super method. The date is maybe required for the line 64?

**Pull request closed without merging**